codalab/codalabtools/deploy/fabfile.py

```diff
@@ -125,7 +125,12 @@ def provision_compute_workers_packages():
     """
     Installs required software packages on a newly provisioned compute worker machine.
     """
-    packages = ('python-crypto libpcre3-dev libpng12-dev libjpeg-dev libmysqlclient-dev uwsgi-plugin-python libsm6 openjdk-7-jre upstart upstart-sysv unixodbc unixodbc-dev')
+    packages = ('python-crypto libpcre3-dev libpng12-dev libjpeg-dev libmysqlclient-dev uwsgi-plugin-python libsm6 openjdk-7-jre unixodbc unixodbc-dev')
+
+    ubuntu_version = run('cat /etc/issue')
```

Comment:
hmm, again in this PR, we have some extra stuff. We're going to be using docker, so I think we can ignore this for now? I'll probably be merging docker into develop once I get tests passing here soon.